specification/components/schemas/PatientSearch.yaml

```diff
@@ -0,0 +1,75 @@
+type: object
+properties:
+  resourceType:
+    type: string
+    description: FHIR resource type.
+    default: "Patient"
+    readOnly: true
+  id:
+    $ref: "NhsNumber.yaml"
+    readOnly: true
+  identifier:
+    $ref: "Identifier.yaml"
+  meta:
+    $ref: "Meta.yaml"
+  name:
+    type: array
+    description: | 
+      List of names associated with the patient.
+      
+      When a patient tagged as `very restricted` is retrieved, all names will be removed from the response.
+    minItems: 1
+    items:
+      $ref: "HumanName.yaml"
+  gender:
+    $ref: "Gender.yaml"
+  birthDate:
+    description: |
+      Date of birth. A date in the format `yyyy-mm-dd`. Due to data quality issues on a small number of patients `yyyy-mm` and `yyyy` format may also be returned.
+      
+      When a patient tagged as `very restricted` is retrieved, birth date will be removed from the response.
+    example: "2010-10-22"
+    type: string
+    format: date
+  deceasedDateTime:
+    description: |
+      Date and time of death, if applicable and known. A datetime in the format `yyyy-mm-ddTHH:MM:SS+HH:MM` or `yyyy-mm-dd`. Due to data quality issues on a small number of patients `yyyy-mm` and `yyyy` format may also be returned.
+      
+      When a patient tagged as `very restricted` is retrieved, death date will be removed from the response.
+    example: "2010-10-22T00:00:00+00:00"
+    type: string
+    format: date-time
+  address:
+    type: array
+    description: | 
+      List of addresses associated with the patient.
+      
+      This will only be fully populated on a retrieval, only a the `home` address will be returned on a search.
+
+      When a patient tagged as `restricted` or `very restricted` is retrieved, all addresses will be removed from the response.
+    items:
+      $ref: "Address.yaml"
+  telecom:
+    type: array
+    description: |
+      List of contact points for the patient; for example, phone numbers or email addresses.
```

Comment:
Same as for name including the systemeffective from/to

Reply:
The business effective dates are included